docs/features.rst

```diff
@@ -25,37 +23,40 @@ Format
         ],
     }
 
-How it works
-````````````````````````
-
-Internally is this parameter used for constructing `WHERE` clause, with *task_ts* and (*task_ts* - *days*) as
-interval borders. In case multiple columns are named, all would be joined with `AND` - therefore only rows which satisfy
-all conditions at the same time will be eligible for further checking.
-
+    # will select data that has `a` column between running time and running time minus 10 days AND `b` column
+    # between running time and running time minus 24 days
+    # then it will do the check against those filtered data only
 
 Miscellaneous
 ````````````````````````
 
- - For backward-compatibility is also supported simple format, which defines only name of the column. In this case default (30) for *days* is used.
+- For backward-compatibility is also supported simple format, which defines only name of the column. In this case default (30) for *days* is used.
 
 .. code-block:: json
 
     {
         "time_filter": "a"
     }
 
-- Parallel to **columns** shortcut for defining multiple rules at once, parameter **separate_time_filters** is available.
-- Each one will be used separately. In case both *separate_time_filters* and *time_filter* are defined, only *separate_time_filters* would be considered.
+- Parameter **separate_time_filters** is available.
+
+  - Each one will be used separately. Meaning, it would actually create X rules and therefore X checks would be done against the data, each filtering the data by passed column. In case both *separate_time_filters* and *time_filter* are defined, only *separate_time_filters* would be considered.
 
 .. code-block:: json
 
     {
+        "name": NOT_NULL,
+        "columns": ["a", "b", "c"],
         "separate_time_filters": [{"column": "c"}, {"column": "d"}]
     }
+    # would check NOT_NULL while filtering by "c" column for last 30 days, write the result.
```

Comment:
Checks `NOT_NULL` while filtering by column `c` and writes the result, then repeats the same check while filtering by column `d` and writes the result as a separate value.